runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -52,6 +52,10 @@
       log("" format "", ##__VA_ARGS__);          \
       }
 
+// From CompositeCache.cpp
+#define UPDATEPTR(ca) (((uint8_t *)(ca)) + (ca)->updateSRP)
+#define SEGUPDATEPTR(ca) (((uint8_t *)(ca)) + (ca)->segmentSRP)
```

Comment:
The reason I can't easily is because the definition in SC code is
https://github.com/eclipse/openj9/blob/04c2621751962dd32b2ff60b8d212d27d5bd953b/runtime/shared_common/CompositeCache.cpp#L69-L71

There are several definitions for `BlockPtr` depending on the file you look at, but I _think_ the one relevant for this macro is  https://github.com/eclipse/openj9/blob/04c2621751962dd32b2ff60b8d212d27d5bd953b/runtime/shared_common/CompositeCacheImpl.hpp#L79 which is typedef'd in a class. Moving the typedef to some j9 header might clash with the other typedefs. We can take a look at what might be involved to expose the SC macros, or maybe even just add public APIs. But I think it would be too much for this PR.